samples/ImplicitFlow/AureliaApp/aurelia_project/generators/attribute.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
@PinpointTownes It appears that we do not need them. I just deleted the directory and it continues to run. Question: would you delete and purge them from the history too or just delete and commit?